tests/test_syntax/blocks/test_html_blocks.py

```diff
@@ -0,0 +1,783 @@
+# -*- coding: utf-8 -*-
+"""
+Python Markdown
+
+A Python implementation of John Gruber's Markdown.
+
+Documentation: https://python-markdown.github.io/
+GitHub: https://github.com/Python-Markdown/markdown/
+PyPI: https://pypi.org/project/Markdown/
+
+Started by Manfred Stienstra (http://www.dwerg.net/).
+Maintained for a few years by Yuri Takhteyev (http://www.freewisdom.org).
+Currently maintained by Waylan Limberg (https://github.com/waylan),
+Dmitry Shachnev (https://github.com/mitya57) and Isaac Muse (https://github.com/facelessuser).
+
+Copyright 2007-2018 The Python Markdown Project (v. 1.7 and later)
+Copyright 2004, 2005, 2006 Yuri Takhteyev (v. 0.2-1.6b)
+Copyright 2004 Manfred Stienstra (the original version)
+
+License: BSD (see LICENSE.md for details).
+"""
+
+from markdown.test_tools import TestCase
+
+
+class TestHTMLBlocks(TestCase):
+
+    def test_raw_paragraph(self):
+        self.assertMarkdownRenders(
+            '<p>A raw paragraph.</p>',
+            '<p>A raw paragraph.</p>'
+        )
+
+    def test_raw_skip_inline_markdown(self):
+        self.assertMarkdownRenders(
+            '<p>A *raw* paragraph.</p>',
+            '<p>A *raw* paragraph.</p>'
+        )
+
+    def test_raw_indent_one_space(self):
+        self.assertMarkdownRenders(
+            ' <p>A *raw* paragraph.</p>',
+            # TODO: reevaluate. This matches strict rules and reference
+            # implementation version 1.0.1 but not 1.0.2b8.
+            '<p><p>A <em>raw</em> paragraph.</p></p>'
```

Comment:
This is easy to fix. We simply allow 0-3 spaces instead of the current 0.

I agree, not producing invalid HTML is a good argument for the change. I asked because this would be a change in behavior for Python-Markdown. But if this change affects existing documents, the affect would be to make what was otherwise invalid HTML valid, so that's not a bad thing.